### ISSUE TITLE
Allow the modular addition of further personal data recording about volunteers (e.g. next of kin, information about health or disability)

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1159]
such data must be held securely: better on our systems with limited access then on individual volunteer manager's personal systems.

**Comment by edickens** [comment]
Just a Notes field will do, provided it is secure and not visible in the Profile by the UserID.  However we need to be careful about holding personal data and opinions under the Data Protection Act.  Guidance in the Help is needed for SCs.

**Comment by PatReynolds** [comment]
Notes field is a good solution. Guidance is in the Security Policy.
Guidance is to store such information (if there is a need to hold it) on
our severs, not scs' computers.

Pat Reynolds
Executive Director, Free UK Genealogy

**Comment by Captainkirkdawson** [comment]
Does not sound like an appropriate use of a Notes field. If we have need to hold secure information it should be stored in an appropriate field and treated accordingly. If we put in the notes field we will need to treat the Notes field securely and not display it except under controlled situations. Not what one would epect to do with a notes field.

**Comment by edickens** [comment]
Agreed.  A new secure field, but it can be free text rather than selecting the reasons.

**Comment by PatReynolds** [comment]
That sounds perfect.

Pat Reynolds
Executive Director, Free UK Genealogy

**Comment by Captainkirkdawson** [comment]
Are any such fields currently proposed? If not this should be closed. What is agreed is a development practice

**Comment by edickens** [comment]
If we do hold this information, then we need to be careful of the Data Protection Act, in particular changes coming shortly.  However, the transcribers themselves may want to supply this personal information so that the SC understands their problems and we can find people who can check out programming Access changes.

**Comment by Captainkirkdawson** [comment]
BUT
Are any such fields currently proposed? If not this should be closed. 
Currently this story has no specific fields so is nothing but philosophy.

**Comment by richpomfret** [comment]
Need to add notes field to be seen by the cc/sc (and higher management) but no one else.

**Comment by Captainkirkdawson** [comment]
Please confirm the requirement is for the addition of a single notes field to be added to the userid_detail model and that it will be populated by an edit or in the original creation

**Comment by PatReynolds** [comment]
Confirmed.
